rlkit/core/rl_algorithm.py

```diff
@@ -265,14 +263,16 @@ def collect_transitions(self, agent, num_samples=1, add_to_enc_buffer=True):
                 agent_info=agent_info,
                 env_info=env_info,
             )
-            if terminal or len(self._current_path_builder) >= self.max_path_length:
+            num_collected += 1
+            if terminal:
```

Comment:
are you sure we don't need to check the max path length on the algorithm side?

Reply:
you're right, I accidentally deleted that! fixed in `de1b5c3`